fdbserver/RestoreApplier.actor.cpp

```diff
@@ -578,4 +610,4 @@ Value applyAtomicOp(Optional<StringRef> existingValue, Value value, MutationRef:
 		ASSERT(false);
 	}
 	return Value();
-}
+}
```

Comment:
Nit: add the new line back.

Reply:
new line back? 
It seems the line is still there with some format change.